protos/google/duration.proto

```diff
@@ -1,117 +0,0 @@
-// Protocol Buffers - Google's data interchange format
```

Comment:
I'm all for pulling directly from Google but probably need some policy around what we should vendor and what we shouldn't

Reply:
in this case, the protobuf compiler distribution often ships with this